cron.py

```diff
@@ -65,6 +67,23 @@ def _check_field(field: str, current_value: int, min_value: int, max_value: int)
     if field == '*':
         return True
     
+    if ',' in field:
+        values = field.split(',')
+        for value in values:
+            if _check_field(value, current_value, min_value, max_value):
+                return True
+        return False
+    
+    if field.startswith('*/') and min_value in [0, 1]:  # Only for minutes and hours fields
```

Comment:
` and min_value in [0, 1]` this part of the check is too hacky. Instead add a `enable_slash: bool` flag and set when calling this for minutes and hours